discord/enums.py

```diff
@@ -575,8 +575,12 @@ def __int__(self) -> int:
 class ComponentType(Enum):
     action_row = 1
     button = 2
-    select = 3
+    string_select = 3
```

Comment:
You shouldn't rename existing enum members since this is a breaking change, at most add an alias for the new name.